src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -449,9 +449,28 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     [estimatedPoolAssetFeesData?.txFeeCryptoBaseUnit, poolAssetFeeAsset?.precision],
   )
 
+  const poolAssetFeeAssetDustAmountCryptoPrecision = useMemo(() => {
+    if (!poolAssetFeeAsset) return 0
+
+    return fromBaseUnit(poolAssetFeeAssetDustAmountCryptoBaseUnit, poolAssetFeeAsset?.precision)
+  }, [poolAssetFeeAssetDustAmountCryptoBaseUnit, poolAssetFeeAsset])
+
+  const poolAssetFeeAssetDustAmountFiatUserCurrency = useMemo(() => {
+    return bnOrZero(poolAssetFeeAssetDustAmountCryptoPrecision).times(
+      poolAssetFeeAssetMarketData.price,
+    )
+  }, [poolAssetFeeAssetMarketData.price, poolAssetFeeAssetDustAmountCryptoPrecision])
+
   const poolAssetGasFeeFiatUserCurrency = useMemo(
```

Comment:
Note, this isn't really a gas fee anymore because it includes dust, but I can't seem to come with another naming that's clearer (`value` isn't it either since this is value + dust). Maybe add a comment to explain this includes dust, so that users get to see the actual value + dust ?

Reply:
Added in [6da3b4d](https://github.com/shapeshift/web/pull/7787/commits/6da3b4d8b8b9161dfb6498096c7c072b68c722a4) and [d4e8386](https://github.com/shapeshift/web/pull/7787/commits/d4e8386331dfb31e9b76b4e366b394e73bde9b2e)